selfdrive/car/ford/carstate.py

```diff
@@ -26,7 +26,10 @@ def update(self, cp, cp_cam):
     # Hybrid variants experience a bug where a message from the PCM sends invalid checksums,
     # we do not support these cars at this time.
     # TrnAin_Tq_Actl and its quality flag are only set on ICE platform variants
-    self.hybrid_platform = cp.vl["VehicleOperatingModes"]["TrnAinTq_D_Qf"] == 0
+    if self.CP.carFingerprint in FORDEV_CAR:
+      self.hybrid_platform = False # BEVs do not experience this
+    else:
+      self.hybrid_platform = cp.vl["VehicleOperatingModes"]["TrnAinTq_D_Qf"] == 0
```

Comment:
```suggestion
    self.hybrid_platform = cp.vl["VehicleOperatingModes"]["TrnAinTq_D_Qf"] == 0
```

Likely fixed in the CAN FD platform, so you can just use CANFD_CAR and get rid of `FORDEV_CAR` set if not used anywhere else. However I would add this exception logic to where `self.hybrid_platform` is used, not here.

Reply:
So, this is the only one I'm not sure about. The hybrid platform flag is set that if it's true, it immediately throws it into dash cam mode. However, on the EVs, this `cp.vl["VehicleOperationModes"]["TrnAinTq_D_Qf"] == 0` is also true, which would cause it to immediately go into dash cam mode. 

Is the suggestion to keep this and then check if it's a hybrid platform AND NOT an EV?

Reply:
I mean to just change the condition where this is used. Makes less sense to change the meaning of `hybrid_platform` for the upstream check.

```python
    if self.CS.hybrid_platform and self.CP.carFingerprint not in CANFD_CAR:
      events.add(car.CarEvent.EventName.startupNoControl)
```

Reply:
Ah, got it. Makes sense. That's what I was thinking, but I put the wrong words in and the incorrect spot. Ty. Updating in branch. Will push changes shortly.